lib/eco/treemanager.py

```diff
@@ -1567,16 +1567,18 @@ def export(self, path=None, run=False, profile=False, source=None, debug=False):
         for p, _, _, _ in self.parsers:
             if p.last_status == False:
                 print("Cannot export a syntactically incorrect grammar")
-                return
+                return False
 
         if str(path).endswith(".aterms"):
             return self.export_aterms(path)
 
         lang = self.parsers[0][2]
         if lang == "Python + Prolog":
             self.export_unipycation(path)
+            return True
         elif lang == "HTML + Python + SQL":
             self.export_html_python_sql(path)
```

Comment:
It works because the export function returns False right at the top if the syntax is incorrect, this is the same for both exporting and running in Eco.